src/MovieDescription.js

```diff
@@ -28,6 +26,23 @@ class MovieDescription extends Component {
       console.log('Something went wrong, please refresh!')
       this.setState({error: 'Something went wrong, please refresh!'})
     })
+    this.fetchTrailerData()
+  }
+
+  fetchTrailerData = () => {
```

Comment:
Gj creating a new function to fetch movie trailer data and invoking it inside another fetch :)